packages/fxa-auth-server/test/local/senders/emails.ts

```diff
@@ -1281,6 +1283,41 @@ const TESTS: [string, any, Record<string, any>?][] = [
     ]],
   ])],
 
+  ['postRecoveryPhoneSignInEmail', new Map<string, Test | any>([
```

Comment:
Looks like some unit test failures in here

Reply:
Yes working on fixing! I'll pop this back into draft until all tests pass.